packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -123,29 +125,35 @@ In general, the test account allows the OIN team to verify that your integration
 
 A configuration guide helps your customers understand how to configure your Okta integration to work with your cloud application.
 
-Provide a configuration guide as part of the OIN submission process. You can format the guide in any format that is accessible through a URL link (such as a webpage, a Google doc, or a PDF). During the OIN verification process, the link to your configuration guide needs to be accessible by the OIN team so that they can check your document for general adherence to the configuration instructions. Ensure that your guide link is public or customer-accessible after your integration is published in the OIN.
+Provide a configuration guide as part of the OIN submission process for all integrations:
 
-Your guide link is available to administrators through the Okta Admin Console.
+* Format the guide so that it's accessible through a URL link (such as a webpage, a Google doc, or a PDF).
+* During the OIN verification process, ensure that the link to your configuration guide is accessible to the OIN team. The OIN team checks your document for general adherence to the configuration instructions.
```

Comment:
should this be `is accessible to the general public` ?

After Ops checks the config guide is good, I don't think they are going back after publishing ensure its public to anyone on the internet.

Reply:
Let me verify with Mohammad Abdullah about this point. This sentence was left in because some ISV might not want their doc public until it's been reviewed and approved.

Reply:
I talked with Mohammad Abdullah: leaving the blurb as-is. some ISVs publish their docs after their app has been published.